p/scripts/main.js

```diff
@@ -536,23 +535,12 @@ function inject_script(name) {
 	document.head.appendChild(script);
 }
 
-function init_sticky_column() {
-	if (!window.$ || !window.$.fn.stick_in_parent) {
-		if (window.console) {
-			console.log('FreshRSS waiting for Sticky-kit…');
-		}
-		window.setTimeout(init_sticky_column, 200);
-		return;
-	}
-	if ($('.toggle_aside').css('display') === 'none') {
-		$('#aside_feed .tree').stick_in_parent({parent:'#aside_feed'});
-	}
-}
-
 function init_column_categories() {
 	if (context.current_view !== 'normal') {
 		return;
 	}
+    
+	$('.dropdown-close a').css('left', ((sidebar.offsetWidth) + "px"));
```

Comment:
If `left` stays at 0, could that simply the JS logic for the dropdown, or is it not advantageous?

At least some parentheses seem to be superfluous :-)
```suggestion
	$('.dropdown-close a').css('left', sidebar.offsetWidth + "px");
```

Reply:
bah, I meant to pull this line out completely when it was decided to put left back to 0.